packages/rest/src/keys.ts

```diff
@@ -62,6 +63,13 @@ export namespace RestBindings {
    * Internal binding key for http-handler
    */
   export const HANDLER = BindingKey.create<HttpHandler>('rest.handler');
+  /**
+   * Binding key for setting and injecting Reject action's error handling
+   * options
+   */
+  export const ERROR_WRITER_OPTIONS = BindingKey.create<ErrorHandlerOptions>(
```

Comment:
Should be `ErrorWriterOptions` after recent changes in the strong-error-handler pull request.